recipes/install.rb

```diff
@@ -26,6 +26,7 @@
 when 'mariadb'
   Array(node['mysqld']['mariadb']['packages']).each { |pkg| package pkg }
```

Comment:
`Unlike with percona, the apt repository for mariadb is not automatically set up in this recipe. While the default recipe makes sure the repository is used, not including it here makes it possible to use distribution packages of mariadb. This is a little dangerous, as the cookbook officially only supports mysql-5.7 and mariadb-10.1`

I'm not sure whether it might be a good idea to include the apt repository here, too.
But as there's no check whether mysql-5.7 is used, we might leave this up to the user. I'm a little concernded when it's possible to easily install a non-supported database version that might lead into problems. Thoughts on this?